app/templates/courses/teacher-classes-view.pug

```diff
@@ -192,7 +192,7 @@ mixin createClassButton
 mixin progressDot(classroom, course, label)
   //- TODO: inefficient. Cache this in the view?
   - courseInstance = view.courseInstances.findWhere({ courseID: course.id, classroomID: classroom.id })
-  if !courseInstance
+  if !courseInstance && view.sharedCourseInstances
```

Comment:
this line is a fix for https://github.com/codecombat/codecombat/pull/6594 , not necessary but i think good to add. so temporay add in this pr so that it would mess the master commits 😂